lib/ipmi_vita.c

```diff
@@ -357,7 +317,7 @@ ipmi_vita_get_vso_capabilities(struct ipmi_intf *intf)
 	tmp = (rsp->data[2] & 0x30) >> 4;
 
 	printf("    Frequency  %skHz\n",
-		tmp == 0 ? "100" : tmp == 1 ? "400" : "RESERVED");
+	       tmp == 0 ? "100" : tmp == 1 ? "400" : "RESERVED");
```

Comment:
Now this is what indeed needed clarification of precedence. Also, I'd format it like this for ease of comprehension:

```
printf("    Frequency %skHz\n",
       (tmp == 0)
       ? "100"
       : (tmp == 1)
         ? "400"
         : "RESERVED"
      );
```
Or better yet, get rid of the ternary operator and use a switch:

```
static const char const *rsvd = "RESERVED";
const char *freq = rsvd;

switch (tmp) {
case 0:
    freq = "100";
    break;
case 1:
    freq = "400";
    break;
}
```

You could then use `rsvd` in other places where `RESERVED` is printed and reduce data duplication a little.